ng-providers/loadingManagerProvider.js

```diff
@@ -242,7 +242,7 @@ angular.module('atlasDemo').provider('loadingManager', ['mainAppProvider', 'volu
 
         var objStructures = atlasStructure.Structure.filter(item => {
             if (Array.isArray(item.sourceSelector)) {
-                return item.sourceSelector.some(selector => /\.obj$/.test(selector.dataSource.source));
+                return item.sourceSelector.some(selector => /\.obj$/.test(selector.dataSource));
```

Comment:
Here we want to list structure using ".obj" files so we can load them later using the OBJLoader.
The correction you provide is wrong as dataSource is an object and the regex can not be used on objects.
To account for the presence of the object, the code should be : test(selector.dataSource && selector.dataSource.source)